moped-database/views/component_arcgis_online_view.sql

```diff
@@ -8,28 +8,36 @@ CREATE OR REPLACE VIEW component_arcgis_online_view AS (
         comp_geography.feature_ids, 
         mpc.component_id, 
         comp_geography.geometry, 
-        comp_geography.line_geometry, 
+        comp_geography.line_geometry,
         comp_geography.signal_ids, 
         council_districts.council_districts, 
+        'placeholder text' as council_districts_searchable,
+        CASE WHEN council_districts IS NULL OR council_districts = '' THEN FALSE
+            ELSE TRUE
+            END AS is_within_city_limits,
         comp_geography.length_feet_total, 
+        round((comp_geography.length_feet_total / 5280::numeric),2) AS length_miles_total,
         mc.component_name, 
         mc.component_subtype, 
         mc.component_name_full, 
-        subcomponents.subcomponents, 
-        work_types.work_types, 
+        'placeholder text' as component_categories,
+        subcomponents.subcomponents as component_subcomponents, 
+        work_types.work_types as component_work_types, 
         component_tags.component_tags, 
-        mpc.description AS component_description, 
-        mpc.is_deleted is_project_component_deleted, 
-        plv.is_deleted is_project_deleted, 
```

Comment:
these are removed because they are always false in the view, right?

Reply:
exactly. removed so as to (hopefully) avoid any confusion.